web/src/pages/onboarding.tsx

```diff
@@ -301,42 +313,27 @@ const OnboardingPage = (props: Props): ReactElement => {
     if (!updateQueue) return;
 
     setRegistrationDimension("Onboarded Guest");
-    if (isAdditionalBusiness) {
-      analytics.event.onboarding_last_step_save_additional_business_button.click.finish_additional_business_onboarding();
-    } else {
-      analytics.event.onboarding_last_step.submit.finish_onboarding();
-    }
 
-    const isRemoteSellerWorker = isRemoteWorkerOrSellerBusiness({
+    isAdditionalBusiness
+      ? analytics.event.onboarding_last_step_save_additional_business_button.click.finish_additional_business_onboarding()
+      : analytics.event.onboarding_last_step.submit.finish_onboarding();
+
+    updateQueue.queueBusiness({
       ...updateQueue.currentBusiness(),
       profileData: newProfileData,
+      onboardingFormProgress: "COMPLETED",
     });
 
-    let newUserData: UserData = {
```

Comment:
newUserData was no longer needed since the updates were being added using the queue; I removed it and continued to added the changes voa the queue